src/ngraph/ngraph_compiler.h

```diff
@@ -186,6 +186,10 @@ class Compiler {
            const mxnet::Context& context);
   // Construct base compiler object with context only
   Compiler(const mxnet::Context& context);
+  // Constructor for use with gluon hybridize
+  Compiler(const nnvm::Graph& graph, const mxnet::Context& context,
+           const std::vector<nnvm::TShape> shapes,
```

Comment:
Should use references on these `const std::vector<>` parameters?

Reply:
!! They should be, not sure how I missed that, thanks.